### ISSUE TITLE
Copy and paste from PDF code results in invalid import statements

### ISSUE THREAD

**Issue by vogella** [asciidoctor/asciidoctor-pdf#832]
If I create a PDF with Asciidoctor and copy it from the PDF and paste it into IntelliJ or Eclipse the import statements will be incorrectly placed.

I assume that the generated PDF is not correct. Is that possible?

For example here is, what I receive in the top node of a paste Java class file. Same result for IntelliJ and Eclipse.

````
package com.vogella.android.databinding;
import
import
import
import
import
android.app.Activity;
android.content.Intent;
android.databinding.DataBindingUtil;
android.os.Bundle;
android.widget.Toast;
import com.vogella.android.databinding.databinding.ActivityMainBinding;
public class MainActivity extends Activity implements MainActivityContract.View {
 ````

**Comment by mojavelinux** [comment]
I've done a fair amount of research on this in the past and what I discovered is that this is an inherent limitation of PDF. The problem is that the PDF format (and the formatting in particular) was designed for print, not for screen—and by screen in this case I'm referring to copy and paste. When the PDF reader lays out the text, it doesn't always do it linearly and doesn't always include spaces. It might absolutely position certain elements depending on what formatting is being applied. And that may cause the reader to using columnar selection.

Long story short, there's really nothing we can do. I'm sure there are some PDF generators that use special tricks to make the text more copy-paste friendly, but we're limited to what Prawn does...and Prawn doesn't provide that feature.

If someone knows some trick we can use, I'd be happy to apply it. But I simply don't know of one.

**Comment by mojavelinux** [comment]
It might be a shot in the dark, but it's possible that the problem is that Prawn doesn't write the space character into the PDF. If it did, the text selection may end up being continuous. But that's really just a guess.

**Comment by mojavelinux** [comment]
I can confirm that making Prawn write the space makes no difference. This is a problem with the reader. Specifically, evince. If I use Adobe Reader, it highlights the lines correctly. So evince is adding some sort of optimization when it sees consecutive lines that begin with the same word, which breaks text selection.

**Comment by vogella** [comment]
Thanks @mojavelinux. Lets close this issue.
